tests/testExecutables.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+import lsst.utils.tests
+
+
+class UtilsBinaryTester(lsst.utils.tests.ExecutablesTestCase):
+    pass
+
+EXECUTABLES = None
+UtilsBinaryTester.create_executable_tests(__file__, EXECUTABLES)
```

Comment:
Why is this here?  After a bit of digging, it seems it needs to be present to get non-`unittest` tests working with our new test system, but the interface is strange.  For example, why is `EXECUTABLES = None`?  That gives the impression that this is a placeholder or somesuch, but I don't think that's right.